src/components/Notice/Notice.jsx

```diff
@@ -103,6 +103,14 @@ function Notice({ children, dismiss, type, className, disableAnimation }) {
     onDismiss: dismiss,
   });
 
+  // We always need to wrap the Notice with AnimationWrapper because the dismiss function
+  // shows the Notice based on the dismissed property. If the animation is disabled the properties
+  // that control the animation are removed from animationProps.
+  if (disableAnimation) {
+    delete animationProps.stageInAnimation;
+    delete animationProps.stageOutAnimation;
+  }
```

Comment:
```suggestion
  const animationWrapperProps = disableAnimation ? {} : animationProps;
```

I'm not sure if this is actually better, but it feels like deleting the individual props specifically could lead to us missing one if someone makes changes to the `useAnimation` hook.

Reply:
The naming and alternative implementation don't really matter to me as much as deleting the props individually.

Reply:
I like the suggestion 👌🏼